spec/amadeus/namespaces.test.js

```diff
@@ -23,7 +23,7 @@ describe('Namespaces', () => {
       expect(amadeus.referenceData.locations).toBeDefined();
       expect(amadeus.referenceData.locations.airports).toBeDefined();
       expect(amadeus.referenceData.locations.pointsOfInterest).toBeDefined();
-      expect(amadeus.referenceData.locations.pointsOfInterest.bySquare).toBeDefined();
+      expect(amadeus.referenceData.locations.pointsOfInterest('XXX').bySquare).toBeDefined();
```

Comment:
Why do you pass a param for a global search bySquare?

Reply:
Fixed

Reply:
Fixed